docs/api/app.md

```diff
@@ -637,6 +637,9 @@ Returns `string` - The current application directory.
     * `~/Library/Application Support` on macOS
   * `userData` The directory for storing your app's configuration files, which by
     default it is the `appData` directory appended with your app's name.
```

Comment:
Let's expand this comment to describe how it differs from `sessionData`, since "user" data is now quite confusing.